great_expectations/datasource/batch_kwargs_generator/query_batch_kwargs_generator.py

```diff
@@ -22,7 +22,34 @@
 
 
 class QueryBatchKwargsGenerator(BatchKwargsGenerator):
-    """Produce query-style batch_kwargs from sql files stored on disk
+    """Produce query-style batch_kwargs from sql files or defined queries.
+
+    Queries can be parameterized using $substitution.
+
+    Example configuration:
+
+      queries:
+        class_name: QueryBatchKwargsGenerator
+        query_store_backend:
+          class_name: TupleFilesystemStoreBackend
+          filepath_suffix: .sql
+          base_directory: queries
+
+    Example query template:
```

Comment:
Would be helpful to link this to the query template name. "Store this in `plugins/queries/movies_by_date.sql`"